### ISSUE TITLE
Unable to sync transaction data from Chargebee

### ISSUE THREAD

**Issue by michaelgaotile** [airbytehq/airbyte#11249]
## Environment
- **Airbyte version**:  0.35.55-alpha
- **OS Version / Instance**: eAWS EC2
- **Deployment**: Docker
- **Source Connector and version**: Chargebee 0.1.11-  Product Catalog 1.0
- **Destination Connector and version**: S3 0.2.10
- **Severity**:  Medium 
- **Step where error happened**:  Sync job 
## Current Behavior
Transaction stream showing no records. API key has read only access - all records. Works for other streams such as customer. 
## Expected Behavior
Expect data to pull transaction data. Data exists on Chargebee WebUI

## Logs
```
2022-03-16 23:26:09 [32mINFO[m i.a.w.w.WorkerRun(call):49 - Executing worker wrapper. Airbyte version: 0.35.55-alpha
2022-03-16 23:26:09 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):105 - Docker volume job log path: /tmp/workspace/1789/0/logs.log
2022-03-16 23:26:09 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):110 - Executing worker wrapper. Airbyte version: 0.35.55-alpha
2022-03-16 23:26:09 [32mINFO[m i.a.w.DefaultReplicationWorker(run):104 - start sync worker. job id: 1789 attempt id: 0
2022-03-16 23:26:09 [32mINFO[m i.a.w.DefaultReplicationWorker(run):116 - configured sync modes: {null.transaction=full_refresh - overwrite}
2022-03-16 23:26:09 [32mINFO[m i.a.w.p.a.DefaultAirbyteDestination(start):69 - Running destination...
2022-03-16 23:26:09 [32mINFO[m i.a.c.i.LineGobbler(voidCall):82 - Checking if airbyte/destination-s3:0.2.10 exists...
2022-03-16 23:26:09 [32mINFO[m i.a.c.i.LineGobbler(voidCall):82 - airbyte/destination-s3:0.2.10 was found locally.
2022-03-16 23:26:09 [32mINFO[m i.a.w.p.DockerProcessFactory(create):106 - Creating docker job ID: 1789
2022-03-16 23:26:09 [32mINFO[m i.a.w.p.DockerProcessFactory(create):158 - Preparing command: docker run --rm --init -i -w /data/1789/0 --log-driver none --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e WORKER_JOB_ATTEMPT=0 -e WORKER_CONNECTOR_IMAGE=airbyte/destination-s3:0.2.10 -e AIRBYTE_ROLE= -e WORKER_ENVIRONMENT=DOCKER -e AIRBYTE_VERSION=0.35.55-alpha -e WORKER_JOB_ID=1789 airbyte/destination-s3:0.2.10 write --config destination_config.json --catalog destination_catalog.json
2022-03-16 23:26:09 [32mINFO[m i.a.c.i.LineGobbler(voidCall):82 - Checking if airbyte/source-chargebee:0.1.9 exists...
2022-03-16 23:26:09 [32mINFO[m i.a.c.i.LineGobbler(voidCall):82 - airbyte/source-chargebee:0.1.9 was found locally.
2022-03-16 23:26:09 [32mINFO[m i.a.w.p.DockerProcessFactory(create):106 - Creating docker job ID: 1789
2022-03-16 23:26:09 [32mINFO[m i.a.w.p.DockerProcessFactory(create):158 - Preparing command: docker run --rm --init -i -w /data/1789/0 --log-driver none --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e WORKER_JOB_ATTEMPT=0 -e WORKER_CONNECTOR_IMAGE=airbyte/source-chargebee:0.1.9 -e AIRBYTE_ROLE= -e WORKER_ENVIRONMENT=DOCKER -e AIRBYTE_VERSION=0.35.55-alpha -e WORKER_JOB_ID=1789 airbyte/source-chargebee:0.1.9 read --config source_config.json --catalog source_catalog.json --state input_state.json
2022-03-16 23:26:09 [32mINFO[m i.a.w.DefaultReplicationWorker(lambda$getDestinationOutputRunnable$6):339 - Destination output thread started.
2022-03-16 23:26:09 [32mINFO[m i.a.w.DefaultReplicationWorker(run):158 - Waiting for source and destination threads to complete.
2022-03-16 23:26:09 [32mINFO[m i.a.w.DefaultReplicationWorker(lambda$getReplicationRunnable$5):279 - Replication thread started.
2022-03-16 23:26:10 [44msource[0m > Starting syncing SourceChargebee
2022-03-16 23:26:10 [44msource[0m > Syncing stream: transaction 
2022-03-16 23:26:10 [43mdestination[0m > SLF4J: Class path contains multiple SLF4J bindings.
2022-03-16 23:26:10 [43mdestination[0m > SLF4J: Found binding in [jar:file:/airbyte/lib/log4j-slf4j-impl-2.17.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2022-03-16 23:26:10 [43mdestination[0m > SLF4J: Found binding in [jar:file:/airbyte/lib/slf4j-log4j12-1.7.25.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2022-03-16 23:26:10 [43mdestination[0m > SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
2022-03-16 23:26:10 [44msource[0m > Giving up for returned HTTP status: `400` and `api_error_code`: `param_wrong_value`
2022-03-16 23:26:10 [44msource[0m > Giving up _send_request(...) after 1 tries (chargebee.api_error.InvalidRequestError: sort_by[asc] : Value created_at specified for this param is incorrect)
2022-03-16 23:26:10 [44msource[0m > sort_by[asc] : Value created_at specified for this param is incorrect
2022-03-16 23:26:10 [44msource[0m > Read 0 records from transaction stream
2022-03-16 23:26:10 [44msource[0m > Finished syncing transaction
2022-03-16 23:26:10 [44msource[0m > SourceChargebee runtimes:
Syncing stream transaction 0:00:00.076967
2022-03-16 23:26:10 [44msource[0m > Finished syncing SourceChargebee
2022-03-16 23:26:11 [43mdestination[0m > SLF4J: Actual binding is of type [org.apache.logging.slf4j.Log4jLoggerFactory]
2022-03-16 23:26:11 [32mINFO[m i.a.w.DefaultReplicationWorker(lambda$getReplicationRunnable$5):305 - Total records read: 0 (0 bytes)
2022-03-16 23:26:11 [32mINFO[m i.a.w.DefaultReplicationWorker(run):163 - One of source or destination thread complete. Waiting on the other.
2022-03-16 23:26:11 [43mdestination[0m > 2022-03-16 23:26:11 [32mINFO[m i.a.i.b.IntegrationCliParser(parseOptions):118 - integration args: {catalog=destination_catalog.json, write=null, config=destination_config.json}
2022-03-16 23:26:11 [43mdestination[0m > 2022-03-16 23:26:11 [32mINFO[m i.a.i.b.IntegrationRunner(runInternal):120 - Running integration: io.airbyte.integrations.destination.s3.S3Destination
2022-03-16 23:26:11 [43mdestination[0m > 2022-03-16 23:26:11 [32mINFO[m i.a.i.b.IntegrationRunner(runInternal):121 - Command: WRITE
2022-03-16 23:26:11 [43mdestination[0m > 2022-03-16 23:26:11 [32mINFO[m i.a.i.b.IntegrationRunner(runInternal):122 - Integration config: IntegrationConfig{command=WRITE, configPath='destination_config.json', catalogPath='destination_catalog.json', statePath='null'}
2022-03-16 23:26:12 [43mdestination[0m > 2022-03-16 23:26:12 [33mWARN[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword examples - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
2022-03-16 23:26:12 [43mdestination[0m > 2022-03-16 23:26:12 [33mWARN[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword airbyte_secret - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
2022-03-16 23:26:12 [43mdestination[0m > 2022-03-16 23:26:12 [32mINFO[m i.a.i.d.s.S3FormatConfigs(getS3FormatConfig):22 - S3 format config: {"format_type":"JSONL","part_size_mb":5}
2022-03-16 23:26:12 [43mdestination[0m > 2022-03-16 23:26:12 [32mINFO[m i.a.i.d.s.j.S3JsonlWriter(<init>):54 - Full S3 path for stream 'transaction': s3://xxxxxx/transaction/2022_03_16_1647473172656_0.jsonl
2022-03-16 23:26:12 [43mdestination[0m > 2022-03-16 23:26:12 [32mINFO[m i.a.i.d.s.u.S3StreamTransferManagerHelper(getDefault):55 - PartSize arg is set to 5 MB
2022-03-16 23:26:13 [43mdestination[0m > 2022-03-16 23:26:13 [32mINFO[m a.m.s.StreamTransferManager(getMultiPartOutputStreams):329 - Initiated multipart upload to xxxxxxx/transaction/2022_03_16_1647473172656_0.jsonl with full ID _9v.7ZY99fS7g6jGp1OQ14PFsPvbcmNbiJdX3WPyuGKPj3nXx1M8DuyZnlJUb7aaeWpqcsQm7wlfwjS2Z877.BZYVDdWf7E.ZymjKc4ztAfGfuyIP8FxCnPtbfM5GpuF
2022-03-16 23:26:13 [43mdestination[0m > 2022-03-16 23:26:13 [32mINFO[m i.a.i.d.s.w.BaseS3Writer(initialize):79 - Overwrite mode
2022-03-16 23:26:13 [43mdestination[0m > 2022-03-16 23:26:13 [32mINFO[m i.a.i.d.s.w.BaseS3Writer(initialize):88 - Purging non-empty output path for stream 'transaction' under OVERWRITE mode...
2022-03-16 23:26:13 [43mdestination[0m > 2022-03-16 23:26:13 [32mINFO[m i.a.i.d.s.w.BaseS3Writer(initialize):92 - Deleted 1 file(s) for stream 'transaction'.
2022-03-16 23:26:13 [43mdestination[0m > 2022-03-16 23:26:13 [33mWARN[m i.a.i.b.IntegrationRunner(watchForOrphanThreads):230 - The main thread is exiting while children non-daemon threads from a connector are still active.
2022-03-16 23:26:13 [43mdestination[0m > Ideally, this situation should not happen...
2022-03-16 23:26:13 [43mdestination[0m > Please check with maintainers if the connector or library code should safely clean up its threads before quitting instead.
2022-03-16 23:26:13 [43mdestination[0m > The main thread is: main (RUNNABLE)
2022-03-16 23:26:13 [43mdestination[0m >  Thread stacktrace: java.base/java.lang.Thread.getStackTrace(Thread.java:1610)
2022-03-16 23:26:13 [43mdestination[0m >         at io.airbyte.integrations.base.IntegrationRunner.dumpThread(IntegrationRunner.java:267)
2022-03-16 23:26:13 [43mdestination[0m >         at io.airbyte.integrations.base.IntegrationRunner.watchForOrphanThreads(IntegrationRunner.java:234)
2022-03-16 23:26:13 [43mdestination[0m >         at io.airbyte.integrations.base.IntegrationRunner.runConsumer(IntegrationRunner.java:189)
2022-03-16 23:26:13 [43mdestination[0m >         at io.airbyte.integrations.base.IntegrationRunner.lambda$runInternal$1(IntegrationRunner.java:162)
2022-03-16 23:26:13 [43mdestination[0m >         at io.airbyte.integrations.base.sentry.AirbyteSentry.executeWithTracing(AirbyteSentry.java:54)
2022-03-16 23:26:13 [43mdestination[0m >         at io.airbyte.integrations.base.sentry.AirbyteSentry.executeWithTracing(AirbyteSentry.java:38)
2022-03-16 23:26:13 [43mdestination[0m >         at io.airbyte.integrations.base.IntegrationRunner.runInternal(IntegrationRunner.java:162)
2022-03-16 23:26:13 [43mdestination[0m >         at io.airbyte.integrations.base.IntegrationRunner.run(IntegrationRunner.java:104)
2022-03-16 23:26:13 [43mdestination[0m >         at io.airbyte.integrations.destination.s3.S3Destination.main(S3Destination.java:48)
2022-03-16 23:26:13 [43mdestination[0m > 2022-03-16 23:26:13 [33mWARN[m i.a.i.b.IntegrationRunner(watchForOrphanThreads):244 - Active non-daemon thread: pool-2-thread-1 (WAITING)
2022-03-16 23:26:13 [43mdestination[0m >  Thread stacktrace: java.base@17.0.1/jdk.internal.misc.Unsafe.park(Native Method)
2022-03-16 23:26:13 [43mdestination[0m >         at java.base@17.0.1/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
2022-03-16 23:26:13 [43mdestination[0m >         at java.base@17.0.1/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
2022-03-16 23:26:13 [43mdestination[0m >         at java.base@17.0.1/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
2022-03-16 23:26:13 [43mdestination[0m >         at java.base@17.0.1/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
2022-03-16 23:26:13 [43mdestination[0m >         at java.base@17.0.1/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
2022-03-16 23:26:13 [43mdestination[0m >         at java.base@17.0.1/java.util.concurrent.ArrayBlockingQueue.take(ArrayBlockingQueue.java:420)
2022-03-16 23:26:13 [43mdestination[0m >         at app//alex.mojaki.s3upload.StreamTransferManager$UploadTask.call(StreamTransferManager.java:484)
2022-03-16 23:26:13 [43mdestination[0m >         at app//alex.mojaki.s3upload.StreamTransferManager$UploadTask.call(StreamTransferManager.java:474)
2022-03-16 23:26:13 [43mdestination[0m >         at java.base@17.0.1/java.util.concurrent.FutureTask.run(FutureTask.java:264)
2022-03-16 23:26:13 [43mdestination[0m >         at java.base@17.0.1/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
2022-03-16 23:26:13 [43mdestination[0m >         at java.base@17.0.1/java.util.concurrent.FutureTask.run(FutureTask.java:264)
2022-03-16 23:26:13 [43mdestination[0m >         at java.base@17.0.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
2022-03-16 23:26:13 [43mdestination[0m >         at java.base@17.0.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
2022-03-16 23:26:13 [43mdestination[0m >         at java.base@17.0.1/java.lang.Thread.run(Thread.java:833)
2022-03-16 23:26:13 [43mdestination[0m > 2022-03-16 23:26:13 [33mWARN[m i.a.i.b.IntegrationRunner(watchForOrphanThreads):244 - Active non-daemon thread: pool-2-thread-2 (BLOCKED)
2022-03-16 23:26:13 [43mdestination[0m >  Thread stacktrace: app//alex.mojaki.s3upload.StreamTransferManager$UploadTask.call(StreamTransferManager.java:483)
2022-03-16 23:26:13 [43mdestination[0m >         at app//alex.mojaki.s3upload.StreamTransferManager$UploadTask.call(StreamTransferManager.java:474)
2022-03-16 23:26:13 [43mdestination[0m >         at java.base@17.0.1/java.util.concurrent.FutureTask.run(FutureTask.java:264)
2022-03-16 23:26:13 [43mdestination[0m >         at java.base@17.0.1/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
2022-03-16 23:26:13 [43mdestination[0m >         at java.base@17.0.1/java.util.concurrent.FutureTask.run(FutureTask.java:264)
2022-03-16 23:26:13 [43mdestination[0m >         at java.base@17.0.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
2022-03-16 23:26:13 [43mdestination[0m >         at java.base@17.0.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
2022-03-16 23:26:13 [43mdestination[0m >         at java.base@17.0.1/java.lang.Thread.run(Thread.java:833)
2022-03-16 23:26:13 [43mdestination[0m > 2022-03-16 23:26:13 [32mINFO[m i.a.i.b.FailureTrackingAirbyteMessageConsumer(close):65 - Airbyte message consumer: succeeded.
2022-03-16 23:26:13 [43mdestination[0m > 2022-03-16 23:26:13 [32mINFO[m i.a.i.d.s.w.BaseS3Writer(close):113 - Uploading remaining data for stream 'transaction'.
2022-03-16 23:26:13 [43mdestination[0m > 2022-03-16 23:26:13 [32mINFO[m a.m.s.MultiPartOutputStream(close):158 - Called close() on [MultipartOutputStream for parts 1 - 10000]
2022-03-16 23:26:13 [43mdestination[0m > 2022-03-16 23:26:13 [32mINFO[m a.m.s.MultiPartOutputStream(close):158 - Called close() on [MultipartOutputStream for parts 1 - 10000]
2022-03-16 23:26:13 [43mdestination[0m > 2022-03-16 23:26:13 [33mWARN[m a.m.s.MultiPartOutputStream(close):160 - [MultipartOutputStream for parts 1 - 10000] is already closed
2022-03-16 23:26:13 [43mdestination[0m > 2022-03-16 23:26:13 [32mINFO[m a.m.s.StreamTransferManager(complete):395 - [Manager uploading to xxxxx/transaction/2022_03_16_1647473172656_0.jsonl with id _9v.7ZY99...tbfM5GpuF]: Completed
2022-03-16 23:26:13 [43mdestination[0m > 2022-03-16 23:26:13 [32mINFO[m i.a.i.d.s.w.BaseS3Writer(close):115 - Upload completed for stream 'transaction'.
2022-03-16 23:26:13 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):56 - JSON schema validation failed. 
errors: $: null found, object expected
2022-03-16 23:26:13 [1;31mERROR[m i.a.w.p.a.DefaultAirbyteStreamFactory(lambda$create$1):70 - Validation failed: null
2022-03-16 23:26:13 [43mdestination[0m > 2022-03-16 23:26:13 [32mINFO[m i.a.i.b.IntegrationRunner(runInternal):168 - Completed integration: io.airbyte.integrations.destination.s3.S3Destination
2022-03-16 23:26:13 [32mINFO[m i.a.w.DefaultReplicationWorker(run):165 - Source and destination threads complete.
2022-03-16 23:26:13 [32mINFO[m i.a.w.DefaultReplicationWorker(run):228 - sync summary: io.airbyte.config.ReplicationAttemptSummary@3e84dc6a[status=completed,recordsSynced=0,bytesSynced=0,startTime=1647473169871,endTime=1647473173469,totalStats=io.airbyte.config.SyncStats@12692658[recordsEmitted=0,bytesEmitted=0,stateMessagesEmitted=0,recordsCommitted=0],streamStats=[]]
2022-03-16 23:26:13 [32mINFO[m i.a.w.DefaultReplicationWorker(run):250 - Source did not output any state messages
2022-03-16 23:26:13 [33mWARN[m i.a.w.DefaultReplicationWorker(run):258 - State capture: No new state, falling back on input state: io.airbyte.config.State@12249ba1[state={}]
2022-03-16 23:26:13 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):131 - Stopping cancellation check scheduling...
2022-03-16 23:26:13 [32mINFO[m i.a.w.t.s.ReplicationActivityImpl(lambda$replicate$1):147 - sync summary: io.airbyte.config.StandardSyncOutput@10852c7d[standardSyncSummary=io.airbyte.config.StandardSyncSummary@142e039e[status=completed,recordsSynced=0,bytesSynced=0,startTime=1647473169871,endTime=1647473173469,totalStats=io.airbyte.config.SyncStats@12692658[recordsEmitted=0,bytesEmitted=0,stateMessagesEmitted=0,recordsCommitted=0],streamStats=[]],state=io.airbyte.config.State@12249ba1[state={}],outputCatalog=io.airbyte.protocol.models.ConfiguredAirbyteCatalog@6880710f[streams=[io.airbyte.protocol.models.ConfiguredAirbyteStream@45eeff05[stream=io.airbyte.protocol.models.AirbyteStream@7c31d501[name=transaction,jsonSchema={"name":"Transaction","type":"object","$schema":"http://json-schema.org/draft-07/schema#","properties":{"id":{"type":["string","null"],"max-length":40},"iin":{"type":["string","null"],"max-length":6},"date":{"type":["integer","null"]},"type":{"enum":["authorization","payment","refund","payment_reversal"],"type":["string","null"]},"last4":{"type":["string","null"],"max-length":4},"amount":{"type":["integer","null"]},"status":{"enum":["in_progress","success","voided","failure","timeout","needs_attention"],"type":["string","null"]},"cn_date":{"type":["integer","null"]},"deleted":{"type":["boolean","null"]},"gateway":{"enum":["chargebee","stripe","wepay","braintree","authorize_net","paypal_pro","pin","eway","eway_rapid","worldpay","balanced_payments","beanstream","bluepay","elavon","first_data_global","hdfc","migs","nmi","ogone","paymill","paypal_payflow_pro","sage_pay","tco","wirecard","amazon_payments","paypal_express_checkout","gocardless","adyen","orbital","moneris_us","moneris","bluesnap","cybersource","vantiv","checkout_com","paypal","ingenico_direct","exact","mollie","not_applicable"],"type":["string","null"]},"cn_total":{"type":["integer","null"]},"txn_date":{"type":["integer","null"]},"cn_status":{"enum":["adjusted","refunded","refund_due","voided"],"type":["string","null"]},"voided_at":{"type":["integer","null"]},"error-text":{"type":["string","null"],"max-length":65000},"error_code":{"type":["string","null"],"max-length":100},"fraud_flag":{"enum":["safe","suspicous","fraudulent"],"type":["string","null"]},"settled_at":{"type":["integer","null"]},"txn_amount":{"type":["integer","null"]},"updated_at":{"type":["integer","null"]},"customer_id":{"type":["string","null"],"max-length":50},"fraud_reason":{"type":["string","null"],"max-length":250},"amount_unused":{"type":["string","null"]},"currency_code":{"type":["string","null"]},"exchange_rate":{"type":["integer","null"]},"id_at_gateway":{"type":["string","null"],"max-length":100},"initiator_type":{"enum":["costumer","merchant"],"type":["string","null"]},"linked_refunds":{"type":["array","null"],"items":{"type":["object","null"],"properties":{"txn_id":{"type":["string","null"],"max-length":40},"txn_status":{"enum":["in_progress","success","voided","failure","timeout","needs_attention"],"type":["string","null"]}}}},"payment_method":{"enum":["card","cash","check"],"type":["string","null"]},"three_d_source":{"type":["boolean","null"]},"linked_invoices":{"type":["array","null"],"items":{"type":["object","null"],"properties":{"applied_at":{"type":["integer","null"]},"invoice_id":{"type":["string","null"],"max-length":50},"invoice_date":{"type":["integer","null"]},"invoice_total":{"type":["integer","null"]},"applied_amount":{"type":["integer","null"]},"invoice_status":{"enum":["paid","posted","payment_due","not_paid","voided","pending"],"type":["string","null"]}}}},"linked_payments":{"type":["array","null"],"items":{"type":["object","null"],"properties":{"id":{"type":["string","null"],"max-length":40},"status":{"enum":["in_progress","success","voided","failure","timeout","needs_attention"],"type":["string","null"]}}}},"refrence_number":{"type":["string","null"],"max-length":100},"refunded_txn_id":{"type":["string","null"],"max-length":40},"subscription_id":{"type":["string","null"],"max-length":50},"resource_version":{"type":["integer","null"]},"amount_capturable":{"type":["integer","null"]},"payment_source_id":{"type":["string","null"],"max-length":40},"gateway_account_id":{"type":["string","null"],"max-length":50},"masked_card_number":{"type":["string","null"],"max-length":20},"linked_credit_notes":{"type":["array","null"],"items":{"type":["object","null"],"properties":{"cn_id":{"type":["string","null"],"max-length":50},"applied_at":{"type":["integer","null"]},"applied_amount":{"type":["integer","null"]},"cn_reason_code":{"enum":["write_off","subscription_change","subscription_cancellation","subscription_pause","chargeback","product_unsatisfactory","service_unsatisfactory","order_change","order_cancellation","waiver","other","fraudulent"],"type":["string","null"]}}}},"authorization_reason":{"enum":["blocking_funds","verification"],"type":["string","null"]},"cn_create_reason_code":{"type":["string","null"],"max-length":100},"merchant_reference_id":{"type":["string","null"],"max-length":500},"cn_reference_invoice_id":{"type":["string","null"],"max-length":50},"reversal_transaction_id":{"type":["string","null"],"max-length":40},"reference_transaction_id":{"type":["string","null"],"max-length":40},"reference_authorization_id":{"type":["string","null"],"max-length":40}}},supportedSyncModes=[full_refresh, incremental],sourceDefinedCursor=true,defaultCursorField=[updated_at],sourceDefinedPrimaryKey=[[id]],namespace=<null>,additionalProperties={}],syncMode=full_refresh,cursorField=[updated_at],destinationSyncMode=overwrite,primaryKey=[[id]],additionalProperties={}]],additionalProperties={}],failures=[]]
2022-03-16 23:26:13 [32mINFO[m i.a.w.t.TemporalUtils(withBackgroundHeartbeat):235 - Stopping temporal heartbeating...
2022-03-16 23:26:13 [32mINFO[m i.a.c.p.ConfigRepository(updateConnectionState):494 - Updating connection 3941a611-2903-4620-94eb-caf8f8f234cd state: io.airbyte.config.State@41a0c642[state={}]
2022-03-16 23:26:13 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):56 - JSON schema validation failed. 
errors: $.compression_codec: is missing but it is required, $.format_type: does not have a value in the enumeration [Avro]
2022-03-16 23:26:13 [32mINFO[m i.a.v.j.JsonSchemaValidator(test):56 - JSON schema validation failed. 
errors: $.flattening: is missing but it is required, $.format_type: does not have a value in the enumeration [CSV]

```
<details>
<summary>LOG</summary> 

```

replace this with
your long log
output here

```

</details>

## Steps to Reproduce
1. Connect Chargebee with a Store Version 1.0, and API with full read access
2. Try syncing transaction data (others may be affected too)
3. No records :(

**Comment by jagannathsrs** [comment]
Having the same issue with Stripe as source and Snowflake as destination

**Comment by marcosmarxm** [comment]
Closed by #14672
